composer.json

```diff
@@ -35,19 +35,22 @@
     "phpunit/phpunit": "^4.0",
     "fabpot/php-cs-fixer": "^1.10",
     "symfony/var-dumper": "^3.0",
-    "symfony/debug": "^3.0"
+    "symfony/debug": "^3.0",
+    "webmozart/path-util": "^2.3"
```

Comment:
Simplify test on kernel root dir. I dont want to make guru regex whereas
human readable library exist

On Tue, 26 Jan 2016 6:59 pm remievaneos <notifications@github.com> wrote:

> In composer.json
> <https://github.com/Evaneos/silex-rest-skeleton/pull/18#discussion_r50873134>
> :
>
> > @@ -35,19 +35,22 @@
> >      "phpunit/phpunit": "^4.0",
> >      "fabpot/php-cs-fixer": "^1.10",
> >      "symfony/var-dumper": "^3.0",
> > -    "symfony/debug": "^3.0"
> > +    "symfony/debug": "^3.0",
> > +    "webmozart/path-util": "^2.3"
>
> what is it used for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Evaneos/silex-rest-skeleton/pull/18/files#r50873134>.
>